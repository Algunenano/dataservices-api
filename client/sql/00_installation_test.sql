-- Install dependencies
CREATE EXTENSION postgis;
CREATE EXTENSION plproxy;

-- Install the extension
CREATE EXTENSION cdb_geocoder_client;

-- Mock the server connection to point to this very test db
SELECT cdb_geocoder_client._config_set('db_server_config', '{"connection_str": "dbname=contrib_regression host=127.0.0.1 user=postgres"}');

-- Mock the server schema
CREATE SCHEMA cdb_geocoder_server;

-- Create a test user to check permissions
DROP ROLE IF EXISTS test_regular_user;
CREATE ROLE test_regular_user;
GRANT publicuser TO test_regular_user;
