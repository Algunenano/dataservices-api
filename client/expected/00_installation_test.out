-- Install dependencies
CREATE EXTENSION postgis;
CREATE EXTENSION plproxy;
-- Install the extension
CREATE EXTENSION cdb_geocoder_client;
